python/orca/example/learn/pytorch/brainMRI/brainMRI.py

```diff
@@ -125,9 +127,7 @@ def model_creator(config):
 
 
 def optim_creator(model, config):
-    optimizer = optim.SGD(model.parameters(),
-                          lr=config.get("lr", 0.001),
-                          momentum=config.get("momentum", 0.9))
+    optimizer = optim.Adam(model.parameters(), lr=config.get("lr", 0.001), )
```

Comment:
why modify this?

Reply:
The source codes use the Adam optimizer.

Reply:
okay